src/main/java/com/gary/backendv2/model/User.java

```diff
@@ -0,0 +1,35 @@
+package com.gary.backendv2.model;
+
+import com.gary.backendv2.model.security.Role;
+import lombok.*;
+
+import javax.persistence.*;
+import java.time.LocalDate;
+import java.util.Set;
+
+@Entity
+@Table(name = "users")
+@Getter
+@Setter
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+public class User {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    Integer userId;
+
+    @Column(nullable = false, unique = true)
+    private String email;
+    @Column(nullable = false)
+    private String password;
+
+    private String firstName;
+    private String lastName;
+    private LocalDate birthDate;
+    private String phoneNumber;
+
+
+    @ManyToMany(fetch = FetchType.EAGER)
+    private Set<Role> roles;
```

Comment:
Nwm, czy nie chcieliśmy by jeden user miał jedną rolę

Reply:
Chcieliśmy

Reply:
to w takim razie trzeba hierarchię ról zaimplementować
bo na przykład jak ktoś by miał rolę dyspozytora to powinien mieć też uprawnienia zwykłego użytkownika itp.
I sprawdzalibyśmy jak wysoko w hierarchii jest dana rola i brali wszystkie uprawnienia od tej roli w dół w hierarchii.
spox mogę zrobić to